### ISSUE TITLE
Default CentOS repo settings are no longer valid.

### ISSUE THREAD

**Issue by madelaney** [deric/puppet-mesos#87]
When trying to bring up a mesos master, and I have set `mesos::install::repo_source` to `mesosphere`. Puppet cannot find the RPM package.

```
Execution of '/usr/bin/rpm -i http://repos.mesosphere.io/el/7/noarch/RPMS/mesosphere-el-repo-7-4.noarch.rpm' returned 1: curl: (22) The requested URL returned error: 404 Not Found error: skipping http://repos.mesosphere.io/el/7/noarch/RPMS/mesosphere-el-repo-7-4.noarch.rpm - transfer failed

change from absent to present failed: Execution of '/usr/bin/rpm -i http://repos.mesosphere.io/el/7/noarch/RPMS/mesosphere-el-repo-7-4.noarch.rpm' returned 1: curl: (22) The requested URL returned error: 404 Not Found error: skipping http://repos.mesosphere.io/el/7/noarch/RPMS/mesosphere-el-repo-7-4.noarch.rpm - transfer failed
```

**Comment by deric** [comment]
Right, that's an issue for Mesosphere. Mesos project officially doesn't support [any binaries](http://mesos.apache.org/downloads/). You can try to file an issue in [Mesosphere JIRA](https://jira.mesosphere.com).

**Comment by deric** [comment]
Resolved in [`v1.1.0`](https://github.com/deric/puppet-mesos/releases/tag/v1.1.0), see #90.
